src/AcceptanceTests/When_using_databus_with_custom_provider.cs

```diff
@@ -66,9 +66,9 @@ public Task Handle(MessageWithLargePayload messageWithLargePayload, IMessageHand
                 readonly Context testContext;
             }
 
-            public class CustomProvider : IProvideBlobServiceClient
+            public class TestableBlobServiceClientProvider  : IProvideBlobServiceClient
```

Comment:
👎 On this change. In cosmos it is called custom as well as in ASP. The acceptance tests exercise the "user level" and from that perspective what matters is that it is possible to provide a custom provider. The fact that it uses the context internally is not relevant enough to be reflected in the name. But if you two prefer it I'll flow along.

Reply:
I'll let Laila break the tie 🙂